src/apps/intel/intel_app.lua

```diff
@@ -97,6 +109,12 @@ function Intel82599:pull ()
    self:add_receive_buffers()
 end
 
+function Intel82599:ingress_packet_drops ()
+   local result = tonumber(self.dev:ingress_packet_drops())
+   counter.set(self.counters["ingress_packet_drops"], result)
```

Comment:
Does `ingress_packet_drops()` already accumulate values? Otherwise you'd want to use `counter.add` rather than `counter.set`.